rbc/heavydb.py

```diff
@@ -413,10 +413,15 @@ def __init__(self,
                  password='HyperInteractive',
                  host='localhost',
                  port=6274,
-                 dbname='omnisci',
+                 dbname='heavyai',
                  **options):
         self.user = user
         self.password = password
+
+        # To-Do: Remove this once we stop supporting HeavyDB 5.9
+        _version = get_heavydb_version()
+        if _version and _version < (6, 0, 0) and dbname == 'heavyai':
+            dbname = 'omnisci'
```

Comment:
In the specific case above, where one specify `dbname = "heavyai"` and heavydb version `< (6, 0)`, will crash RBC. The `heavyai` database only exist after the rebranding. 

FYI, I had to change this to run the notebook tests after the rebranding.

Reply:
The crash occurs because the default database names changed from 5.9 to 6.0.
Otherwise, one can create new databases with arbitrary names and then connect to these by using dbname argument.